sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -689,11 +673,471 @@ export enum KnownAnalyzerNames {
     ZhHantMicrosoft = "zh-Hant.microsoft"
 }
 
+// @public
+export const enum KnownApiVersion20200630 {
+    TwoThousandTwenty0630 = "2020-06-30"
```

Comment:
this is a pretty insane enum name, why are we trying to convert a date to a weird mix of english and numbers?

Reply:
We are not exporting this anymore